Whiskey/Tasks/Publish-WhiskeyBuildMasterPackage.ps1

```diff
@@ -15,19 +15,48 @@ function Publish-WhiskeyBuildMasterPackage
 

```

Comment:
Can you add documentation for the task? We've been negligent in our docs and I'd like us to start creating docs and/or updating them whenever we touch a function.

Reply:
Yup. Added.